cli/flox/src/commands/publish.rs

```diff
@@ -72,16 +73,8 @@ impl Publish {
         }
         .or_else(|e| bail!(e.to_string()))?;
```

Comment:
Would this line above be changed in the same way? What then happens if called with a `RemoteEnvironment`? Will we need to be more careful about implicitly accepting all environments?
```rust
        let env_metadata =
            check_environment_metadata(&flox, &mut env).or_else(|e| bail!(e.to_string()))?;
```

Reply:
Added an explicit bail. I think that's clear enough